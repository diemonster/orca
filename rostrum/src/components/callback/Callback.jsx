import React from 'react';

function Callback() {
  return (
    <div className="container">
      <h4>Loading...</h4>
    </div>
  );
}

export default Callback;
