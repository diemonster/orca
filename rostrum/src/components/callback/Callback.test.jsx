import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Callback from './Callback';

Enzyme.configure({ adapter: new Adapter() });

describe('components', () => {
  describe('Callback', () => {
    it('should render itself', () => {
      const callback = shallow(<Callback />);

      expect(callback.find('div').hasClass('container')).toBe(true);
      expect(callback.find('h4').text()).toEqual('Loading...');
    });
  });
});
