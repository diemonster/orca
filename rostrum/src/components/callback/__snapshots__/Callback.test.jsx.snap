// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Callback should render itself 1`] = `
<div
  className="container"
>
  <h4>
    Loading...
  </h4>
</div>
`;
